src/main/kotlin/no/nav/familie/ef/sak/barn/BarnService.kt

```diff
@@ -291,4 +291,23 @@ class BarnService(
     fun hentBehandlingBarnForBarnIder(barnId: List<UUID>): List<BehandlingBarn> {
         return barnRepository.findAllByIdOrThrow(barnId.toSet()) { it.id }
     }
+
+    /**
+     * Kan strengt tatt kun brukes for barnetilsyn då den er avhengig av at personIdent finnes, som skal finnes på alle barn til barnetilsyn
```

Comment:
Hvis denne kun er for barnetilsyn - Kunne lagt inn barnetilsyn i navn på metode (evt parameternavn barnetilsynBehandlingsId), eller enda strengere validere at behandling er barnetilsyn?

Reply:
Enig, endret til `kobleBarnForBarnetilsyn`